content/docs/tutorials/analyze_pages_and_fix_errors.md

```diff
@@ -109,29 +109,36 @@ Snapshots can tell you if:
 
 In Apify scrapers (**Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)), **Cheerio Scraper** ([apify/cheerio-scraper](https://apify.com/apify/cheerio-scraper)) and **Puppeteer Scraper** ([apify/puppeteer-scraper](https://apify.com/apify/puppeteer-scraper))), you can use their built-in `context.saveSnapshot()` function. Once called, it saves a screenshot and HTML into the run's **key-value store**.
 
-When **building your own actors** with [Puppeteer](https://pptr.dev) or the the [Apify SDK](https://sdk.apify.com) package, you can use the powerful `utils.puppeteer.saveSnapshot()` [function](https://sdk.apify.com/docs/api/puppeteer#puppeteersavesnapshot). It allows you name the screenshot, so you can identify it later.
+When **building your own actors** with [Puppeteer](https://pptr.dev) or the [Apify SDK](https://sdk.apify.com) package, you can use the powerful `utils.puppeteer.saveSnapshot()` [function](https://sdk.apify.com/docs/api/puppeteer#puppeteersavesnapshot). It allows you name the screenshot, so you can identify it later.
```

Comment:
There aren't any utils.puppeteer in SDK v3 I believe so this is misleading. Needs to reference Crawlee

Reply:
I did not touch the references at all yet, this PR is about snippets only, looks like I just looked at this `the` and did not like it and just removed =D

Reply:
It is actually there, but only in the metapackage:

https://github.com/apify/crawlee/blob/master/packages/crawlee/src/index.ts#L18-L25